src/release-pr.ts

```diff
@@ -504,64 +504,19 @@ export class ReleasePR {
     prefix?: string,
     preRelease = false
   ): Promise<GitHubTag | undefined> {
+    const branchPrefix = prefix?.endsWith('-')
+      ? prefix.replace(/-$/, '')
+      : prefix;
     // only look at the last 250 or so commits to find the latest tag - we
     // don't want to scan the entire repository history if this repo has never
     // been released
-    const pull = await this.findMergedReleasePR([], prefix, preRelease, 250);
-    if (!pull) return await this.gh.latestTagFallback(prefix, preRelease);
-
-    // TODO: shouldn't need to do this twice
-    const branchName = BranchName.parse(pull.headRefName)!;
-    const version = await this.detectReleaseVersion(pull, branchName);
-    const normalizedVersion = semver.valid(version)!;
```

Comment:
This is now no longer necessary with the async iterator - don't need to return the pull request and parse this information again.